src/XMakeBuildEngine/Definition/ToolsetReader.cs

```diff
@@ -157,89 +157,86 @@ ToolsetDefinitionLocations locations
             {
 #if FEATURE_WIN32_REGISTRY
                 if (NativeMethodsShared.IsWindows || registryReader != null)
```

Comment:
Since, this check is indeed required (https://github.com/Microsoft/msbuild/pull/693#discussion_r68432809), the only thing required to be fixed here is the opening brace for this `if` block. And just indent the corresponding code. Something like - https://gist.github.com/radical/f085ffb25f57b6fc179f785e4a47b930 .